sdk/dotnet/sdk/Dagger.SDK.SourceGenerator.Tests/Dagger.SDK.SourceGenerator.Tests.csproj

```diff
@@ -7,14 +7,15 @@
         <TestingExtensionsProfile>Default</TestingExtensionsProfile>
         <TestingPlatformDotnetTestSupport>true</TestingPlatformDotnetTestSupport>
         <TestingPlatformShowTestsFailure>true</TestingPlatformShowTestsFailure>
+        <LangVersion>default</LangVersion>
     </PropertyGroup>
 
     <ItemGroup>
-        <ProjectReference Include="..\Dagger.SDK.SourceGenerator\Dagger.SDK.SourceGenerator.csproj"/>
+        <ProjectReference Include="..\Dagger.SDK.SourceGenerator\Dagger.SDK.SourceGenerator.csproj" />
     </ItemGroup>
 
     <ItemGroup>
-        <PackageReference Include="Microsoft.CodeAnalysis.CSharp.SourceGenerators.Testing.MSTest" Version="1.1.1"/>
```

Comment:
This has been deprecated, Microsoft recommend move to more generic version (without .MSTest)